### DISCUSSION TITLE
Where can I see previous change logs (release notes)?

### DISCUSSION THREAD

**Discussion by drcjones** [aclap-dev/video-downloadhelper#124] [Help and Q&A]
### Your question

I have searched all over online, including these GitHub pages and the Google group, yet cannot seem to find previous change logs (release notes) for Video DownloadHelper. I'm sorry if it is glaringly obvious, but could you point me to them? 

### Addon & CoApp version and details

All versions

**Comment by Petervos2018** [comment]
https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/versions/ for firefox

**Comment by paulrouget** [comment]
We are also doing announcements here: https://github.com/aclap-dev/video-downloadhelper/discussions/categories/announcements

**Comment by GrampaWildWilly** [comment]
I think you're flat out of luck for Chrome & Edge.  Chrome is housed on the Google Chrome web store & Edge is housed on the Microsoft Edge web store.  VDH for those browsers is housed on those web stores, both the general availability & beta versions for each browser.  Neither Google not Microsoft appears to keep archives of backlevels of anything, entirely unlike Mozilla for Firefox.  Perhaps there is something on the VDH web site www.downloadhelper.net.  I'm not aware of it but that could just mean I'm ignorant & it is there after all.  From this point forward, it seems the announcements Paul mentioned may serve the purpose.  But that doesn't help you for versions prior to now.
